prime-router/src/main/kotlin/azure/SubmissionFunction.kt

```diff
@@ -113,62 +114,60 @@ class SubmissionFunction(
         }
     }
 
-    @FunctionName("getSubmissionHistory")
```

Comment:
This Function is no longer needed.   getReportHistory now handles both actionId and reportId queries.